src/FDevs/ArticleBundle/Resources/views/Default/articleBlock.html.twig

```diff
@@ -1,7 +1,9 @@
 <article>
     <header>
         <h2>
-            <a href="{{ path('f_devs_article_article',{"slug":article.id,'username': app.request.get('username')}) }}">{{ article.title }}</a>
+            <a href="{{ path('f_devs_article_article',{"slug":article.id,'username': app.request.get('username')}) }}">
+                {{ article.title|raw }}
```

Comment:
allow only those tags which use. 
Example `|striptags('<em></em>')` or create custom filter or create custom tpl for the search